### PR TITLE
Aged Brie

### PR DESCRIPTION
**Pull request by anapred345**
tests for Aged Brie
implemented a separate if for the Aged Brie item

### CODE REVIEW

**Comment by anapred345** [comment]
I updated the tests because I realised I didn't test and edge case:
- The sell by date passes and the Quality = 49. Because the sell by date passes, quality will double increase => quality = 49+2 = 51.

File: app/gilded-rose.ts [hunk 1]
```diff
@@ -19,51 +19,49 @@ export class GildedRose {
...
-                    if (this.items[i].name != 'Sulfuras, Hand of Ragnaros') {
-                        this.items[i].quality = this.items[i].quality - 1
+            if(this.items[i].name != 'Sulfuras, Hand of Ragnaros'){
+                if(this.items[i].name === 'Aged Brie'){
+                    if (this.items[i].quality < 50) {
+                        this.items[i].quality = this.items[i].quality + 1
+                        if (this.items[i].sellIn <= 0)
+                            this.items[i].quality = this.items[i].quality + 1
+                        if(this.items[i].quality > 50)
+                            this.items[i].quality = 50;
```
Comment (DanielaNegrut) [inline app/gilded-rose.ts:29]: This can be extracted into a separate function called for example `updateAgedBrie` so it is much cleaner 😄

**Review by DanielaNegrut — APPROVED** [review]
Overall looks good
